Analyzer/plugins/Analyzer.cc

```diff
@@ -198,7 +203,7 @@ Analyzer::Analyzer(const edm::ParameterSet& iConfig)
   else
     trackerCorrector.TrackerGains = nullptr;
 
-  moduleGeom::loadGeometry(geometry_);
```

Comment:
This part is not used anymore?

Reply:
Yep, I take this from the tTopo now (good for all Phase-0,Phase-1,Phase-2)